pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -271,14 +269,13 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 		updateCleanupPolicy = kubectl.RenameRollingUpdateCleanupPolicy
 	}
 	config := &kubectl.RollingUpdaterConfig{
-		Out:            out,
-		OldRc:          oldRc,
-		NewRc:          newRc,
-		UpdatePeriod:   period,
-		Interval:       interval,
-		Timeout:        timeout,
-		CleanupPolicy:  updateCleanupPolicy,
-		UpdateAcceptor: kubectl.DefaultUpdateAcceptor,
+		Out:           out,
+		OldRc:         oldRc,
+		NewRc:         newRc,
+		UpdatePeriod:  period,
```

Comment:
Yeah, I'll be implementing `IntOrString` now. The way defaults are currently enforced is by setting the percentage values to 0. With `IntOrString` I'll update the callers to use absolute 1/1 and reject 0/0 as invalid.